sdk/tests/sdk-tests/ik-upload.component.spec.ts

```diff
@@ -597,4 +623,17 @@ describe("IkUploadComponent", () => {
     comp.handleUploadResponse(undefined, 'success', options, xhr, progressCb);
     expect(hasTrackedProgress).toBeTruthy();
   });
+
+  it("abort should have been called when when upload.abort is invoked", () => {
+    component.fileName = 'dummy-file-name';
+    fixture.detectChanges();
+    const input = fixture.nativeElement.children[0];
+    input.dispatchEvent(new Event('change'));
+    fixture.detectChanges();
+    spyOn(window.console, 'log');
+    component.xhr = new XMLHttpRequest();
+    component.abort();
+    // There would have been a `Upload aborted` log printed
+    expect(window.console.log).toHaveBeenCalled();
```

Comment:
Done

Reply:
But this is decreasing code coverage since the test case completes after the function is triggered so statements inside the function are not covered. Decreased by around 0.8-0.9%